projects/js-packages/shared-extension-utils/src/get-jetpack-data.js

```diff
@@ -5,6 +5,11 @@ import { get } from 'lodash';
 
 export const JETPACK_DATA_PATH = 'Jetpack_Editor_Initial_State';
 
+/**
+ * Retrieves Jetpack editor state
+ *
+ * @returns {object|null}Object The Jetpack Editor State.
+ */
 export default function getJetpackData() {
 	return get( 'object' === typeof window ? window : null, [ JETPACK_DATA_PATH ], null );
```

Comment:
I wonder if we could use:

```suggestion
	return 'object' === typeof window ? window?.[ JETPACK_DATA_PATH ] ?? null : null;
```

and remove the lodash dependency

Reply:
That should work. 

In fact, I believe '`object' === typeof window` check will not be required with the conditional` window?.[ JETPACK_DATA_PATH ]`

Making this change

Reply:
> I believe 'object' === typeof window check will not be required 

Yeah I thought that too, but in my tests if you do something like `undefVar?.property` you get an error that `undefVar` is undefined. If this is always called in the browser and not server side, then I think it's probably ok to remove the object type check.